src/odemis/acq/stream/_sync.py

```diff
@@ -1753,10 +2132,10 @@ def _runAcquisitionScanStage(self, future):
                         cor_pos = raw_pos[0] + strans[0], raw_pos[1] + strans[1]
                         logging.debug("Updating pixel pos from %s to %s", raw_pos, cor_pos)
 
-                    # In practice, for the e-beam data, it's only useful to
-                    # update the metadata for the first pixel.
-                    for adas in self._acq_data:
-                        adas[-1].metadata[MD_POS] = cor_pos
+                    # # In practice, for the e-beam data, it's only useful to
+                    # # update the metadata for the first pixel.
+                    # for adas in self._acq_data:
+                    #     adas[-1].metadata[MD_POS] = cor_pos
```

Comment:
is this commented on purpose? If yes, could you add a comment to explain when it can be used.